src/main/resources/templates/tutorView.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE HTML>
+<html xmlns:th="http://www.thymeleaf.org" lang="en">
+
+<head>
+  <div th:replace="bootstrap/title.html"></div>
+  <div th:replace="bootstrap/bootstrap_head.html"></div>
+</head>
+
+<body>
+  <div class="container">
+    <div th:replace="bootstrap/bootstrap_nav_header.html"></div>
+
+    <div class="row">
+      <div class="col-md-6">
+
+        <h2 class="my-5">Tutor Page</h2>
+        <h4>CSV must be in the following format:</h4>
+        <pre>
+email,lastName,firstName
+richert@ucsb.edu,Wang,Richert
+dimirza@ucsb.cs.edu,Mirza,Diba
+phtcon@ucsb.edu,Conrad,Phill
+        </pre>
+        <!---<br/>
+        <form action="/tutors/upload" method="post" enctype="multipart/form-data">
+          <input type="hidden" th:name="${_csrf.parameterName}" th:value="${_csrf.token}">
+          <input type="file" name="csv" id="csv" accept="text/csv">
+          <input type="submit" class="btn btn-primary" value="Submit">
+        </form>
```

Comment:
Html shouldn't be raw copy of Tutor page. Either add mvp page that has just title: "Tutor View Page". If possible query tutor assignments table using tutor's email and show list of courses assigned this particular tutor. Or you could simply add this as another story on the kanban board.